assets/src/api.ts

```diff
@@ -60,20 +60,14 @@ export const apiCall = <T>({
 }: {
   url: string
   parser: (data: any) => T
-  defaultResult?: T
+  defaultResult: T
   fetchArgs?: RequestInit
 }): Promise<T> =>
   fetch(url, fetchArgs)
     .then(checkResponseStatus)
     .then((response) => parseJson(response) as any)
     .then(({ data: data }: { data: any }) => parser(data))
-    .catch((error) => {
```

Comment:
Maybe we could use [Sentry.captureException](https://docs.sentry.io/platforms/javascript/guides/react/usage/#capturing-errors)? I haven't used it myself before, but see that glides & dotcom have a few references to it

Reply:
Interesting - it sounds like just plain passing the error object into `captureException` will record it just local a normal exception, and part of what we're hoping to accomplish here is reducing alert fatigue in Sentry. However, I also note that there's `captureMessage`, but the documentation doesn't really explain how that shows up in Sentry. So I'm not sure either of those is _quite_ appropriate but it's good food for thought. 🤔

Reply:
I ended up doing sort of the inverse in 06b75de4, I figured we don't want to capture all errors but one type of error we specifically do want to capture is when the frontend gets an invalid API response. In 06b75de4 I made it so that `checkedApiCall` still specifically calls `Sentry.capturreException` on a `StructError` but otherwise ignores it, and always returns the default value on error either way. Of course his doesn't work for regular, un-checked `apiCall` but those already have the issue that bad data won't trigger an exception until later.

Reply:
That seems like a reasonable approach to me! I guess I haven't really thought through the range/variety of errors we could get at this point in the flow. But StructError seems like an important one to be catching!